src/main/drivers/pitotmeter/pitotmeter_ms4525.c

```diff
@@ -74,6 +74,13 @@ static bool ms4525_read(pitotDev_t * pitot)
     dP_raw2 = 0x3FFF & ((rxbuf2[0] << 8) + rxbuf2[1]);
     dT_raw2 = (0xFFE0 & ((rxbuf2[2] << 8) + rxbuf2[3])) >> 5;
 
+    // reject any values that are the absolute minimum or maximums these
```

Comment:
Are these changes needed for the new sensor, or an unrelated change made during development?

Reply:
This is additional safeguard taken directly from ArduPlane MS4525 driver (AP_Airspeed_MS4525.cpp line 180) to ensure that read values are correct. I guess it rarely is hit. And yes, it is unrelated.